core/src/main/java/com/beepiz/bluetooth/gattcoroutines/experimental/extensions/GattService.kt

```diff
@@ -4,9 +4,49 @@ import android.bluetooth.BluetoothGattService
 import android.os.Build.VERSION_CODES.JELLY_BEAN_MR2
 import android.support.annotation.RequiresApi
 import com.beepiz.bluetooth.gattcoroutines.experimental.BGC
+import com.beepiz.bluetooth.gattcoroutines.experimental.BGD
+import com.beepiz.bluetooth.gattcoroutines.experimental.GattConnection
 import java.util.*
```

Comment:
@LouisCAD Although not part of this PR, it is generally advised against using `*` for imports.

> Wildcard imports (of any type) are not allowed.

https://developer.android.com/kotlin/style-guide

Perhaps worth changing outside of this PR?

Reply:
@twyatt This implies changing the IDE to not auto use wildcard import for this specific `java.util` package, but is really out of the scope of this PR. Feel free to create an issue for this if you want.